lib/Queries/ReadQueries.js

```diff
@@ -197,7 +192,7 @@ class ReadQueries extends Queries {
                 1,
                 reference.perPage(),
                 typeof(filters) === 'function' ? filters(search) : filters,
-                null,
+                reference,
```

Comment:
`getRawValues` method take an object wich is a list of field indexed by name.

IMO, this will be ignore.